lib/wiki_course_edits.rb

```diff
@@ -117,12 +122,14 @@ def repost_with_sanitized_links(wiki_title, wiki_text, summary, blacklist)
 
   def add_course_template_to_instructor_userpage(instructor)
     user_page = "User:#{instructor.username}"
-    template = "{{course instructor|course = [[#{@course.wiki_title}]] }}\n"
+    instructor_template_key = @templates['templates']['instructor']
```

Comment:
Let's extract all these _key variables into a method, `template_name(key)`, which will raise an error if the key is missing.